.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Please complete the following sections as necessary. -->
+
+## Description
+
+<!-- Summary of the changes, related issue, relevant motivation, and context -->
+
+### Ticket
+
+<!-- Link to ticket in pivotal -->
+
+### Approach
+
+<!-- Any changed dependencies, e.g. requires an install/update/migration, etc. -->
+
+### Notes
+
```

Comment:
So we also had a **steps to test** section, is this something we want to consider here? I feel like with so many new people (Me, JP, who ever ends up replacing Anne) on the team it might be good to have specific instructions about how tot test unintuitive functionality?

Reply:
What does this mean exactly? Do you mean test like unit test? Or test like go-into-the-dev-site?

Reply:
The latter, like a set of go here on the site and do these things to get this to show up most easily. (in case they were unfamiliar with how to get the feature to show up exactly)

Reply:
Do you pull people's code and run it when you're reviewing PRs? This seems like something that'd be useful for the reviewers in the Tracker story, but I don't do that during code review

Reply:
Historically, yes. QA is everyone's responsibility–anyone conducting a review (code, design, product, etc.) is a QA checkpoint. Otherwise I'm putting my approval on something that I can't guarantee works.

Reply:
That's interesting, I'd love to have a larger conversation regarding that expectation. I feel like we need to be able to trust each other. If you say "you can test this story by logging in as Dakota, opening the Formation task, and seeing the new field" then I don't feel the need to perform those steps again - you being the engineer who coded it, you already did so. I'm not there to perform product testing to validate the AC, or design testing to validate the look - you did that before you delivered the story. Shouldn't I be able to trust you that if you say "the field is there" then it actually is? 

If something is complex and I don't understand it, I'll pull it for my own benefit to try to poke around more. But it feels like a lot to ask to ask every PR reviewer to also be a design reviewer and a product tester.